scripts/lernaAliasNorthstar.js

```diff
@@ -1,28 +1,42 @@
 const lernaAlias = require('lerna-alias');
+const findGitRoot = require('./monorepo/findGitRoot');
+const path = require('path');
 
 // northstar packages should pull these from npm, not the repo
 const excludedPackages = ['@fluentui/date-time-utilities', '@fluentui/dom-utilities', '@fluentui/react-compose'];
 
+function deleteNorthstarAliases(aliases) {
```

Comment:
I'm confused, the whole point of this file was to be lerna-alias *specifically for* things under packages/fluentui.

Reply:
yes, I am trying to remove the aliases by pointing the entrypoints to raw src code, until publish time

Reply:
Since I haven't modified the entrypoints for packages outside of packages/fluentui I still need those aliases

Reply:
Ah okay, that makes sense. Can you add a comment about that?

Reply:
that's a good point ! I'll do that